department-of-veterans-affairs/abd-vro
# MCP-2519 Python Container Cleanup

## What was the problem?

They Python containers had copies of `logging_setup` and `main_consumer` which weren't needed.

Associated tickets or Slack threads:

- [MCP-2519](https://amida.atlassian.net/browse/MCP-2519)

## How does this fix it?[^1]

Remove the duplicate copies and have 1 set of these files in the main `service-python` directory which the `Dockerfile` will copy into each container when building the image

## How to test this PR
- Verify that each container starts and check that E2E passes


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
